src/Checkpoints.jl

```diff
@@ -74,7 +74,16 @@ available() = collect(keys(CHECKPOINTS))
 
 Returns a vector of all enabled ([`config`](@ref)ured) checkpoints.
 """
-enabled_checkpoints() = filter(k -> CHECKPOINTS[k] !== nothing, available())
+enabled_checkpoints() = filter(k -> CHECKPOINTS[k] isa Handler, available())
```

Comment:
It seems a bit odd that we can `config` a deprecated checkpoint and have it not show up in the enabled checkpoints. Maybe add a note to the docstring?